integration-cli/check_test.go

```diff
@@ -211,7 +211,7 @@ func (s *DockerSwarmSuite) AddDaemon(c *check.C, joinSwarm, manager bool) *Swarm
 		port:   defaultSwarmPort + s.portIndex,
 	}
 	d.listenAddr = fmt.Sprintf("0.0.0.0:%d", d.port)
-	err := d.StartWithBusybox("--iptables=false") // avoid networking conflicts
+	err := d.StartWithBusybox("--iptables=false", "--swarm-default-advertise-addr=lo") // avoid networking conflicts
```

Comment:
Won't this change to use `loopback` will result in `TestApiSwarmForceNewCluster` to fail ? , especially because of the way `AddDaemon` is constructed. Both the daemons will advertise 127.0.0.1 as the address to reach and the 2nd daemon (manager) tries to join the first daemon, it will try to connect to its own loopback address ?